benchmarks/lib/build.gradle.kts

```diff
@@ -47,9 +47,11 @@ jmh {
         includes.add(includeBenchmark)
     }
     fork.set(1)
-    benchmarkMode.add("thrpt")
```

Comment:
Many of the benchmarks assume a single threaded execution. It's best not to change this global setting and just modify it at a benchmark level.

Reply:
Fixed by new commit: I've removed the timeUnit and benchmarkMode completely so they default to ops/s. And I've added annotations in the Random benchmarks to force ns/op there.